lib/charms/opensearch/v0/opensearch_relation_provider.py

```diff
@@ -59,7 +59,7 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 3
```

Comment:
Is this correct?

Reply:
Yes - the Libpatch on charmhub is 1, i'm a bit unsure why we had 3 set in the first place